dist2src/core.py

```diff
@@ -596,6 +628,8 @@ def add_packit_config(self, upstream_ref: str, commit: bool = False):
                 },
             ],
         }
+        self.set_sources_in_packit_config(config)
```

Comment:
Something like: 

```suggestion
            ],
            "sources": self.get_sources()
        }
```

would be probably easier to read, and tell what the content of `.packit.yaml` is going to be.

Reply:
oh, yeah, that's nice